docs/guide/connecting.asciidoc

```diff
@@ -3,58 +3,155 @@
 
 This page contains the information you need to connect the Client with {es}.
 
+[discrete]
+[[connect-ec]]
+=== Connecting to Elastic Cloud
+
+https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html[Elastic Cloud] is the easiest way to get started with Elasticsearch. When connecting to Elastic Cloud with the Python Elasticsearch client you should always use the `cloud_id` parameter to connect. You can find this value within the "Manage Deployment" page after you've created a cluster (look in the top-left if you're in Kibana).
+
+When you use the `cloud_id` parameter your client will be automatically configured for optimal use with Elastic Cloud including HTTPS and HTTP compression.
+
+```python
+from elasticsearch import Elasticsearch
+
+# Generated by Elasticsearch on creation of deployment
+ELASTIC_PASSWORD = "<password>"
```

Comment:
Should we suggest reading the password from an environment variable or is that out of scope?